packages/vaex-arrow/vaex_arrow/convert.py

```diff
@@ -0,0 +1,34 @@
+import pyarrow
+import numpy as np
+from .dataset import DatasetArrow
+
+def arrow_array_from_numpy_array(array):
+    dtype = array.dtype
+    mask = None
+    if np.ma.isMaskedArray(array):
```

Comment:
We should support this natively in Arrow. Can you open a JIRA there?

Reply:
I think so too, I wanted to move fast, but I think many of these features should go into pyarrow.

Reply:
Feel free to just hammer at us with JIRAs. We're happy for them; we're even happier when we get PRs for them but just JIRAs are also helpful to see what the missing pieces are that people actually need.